src/main/java/org/alliancegenome/mati/entity/SubdomainEntity.java

```diff
@@ -0,0 +1,32 @@
+package org.alliancegenome.mati.entity;
+
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotEmpty;
+
+@Entity(name = "Subdomain")
+@Table(name = "subdomain")
+@Data
+public class SubdomainEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "code")
+    @NotEmpty
+    private String code;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "description")
```

Comment:
I would suggest not using @Column if they are going to be the same name as the column. We have worked on several major projects where this turns into a hassle because the variables in the code do not match the fields in the database. So I would suggest leaving them out.